eureka-common/src/main/java/edu/emory/cci/aiw/cvrg/eureka/common/entity/ExtendedPhenotype.java

```diff
@@ -72,6 +74,12 @@
 
 	private Integer minDuration;
 
+        @OneToMany(cascade = CascadeType.REMOVE,mappedBy = "lhsExtendedPhenotype")   
```

Comment:
The change in ExtenededPhenotype.java is necessary.
We do have the foreign key from Relations to ExtendedPhenotype already, but it is unidirectional relationship from Relations to ExtendedPhenotype. And our design now is changed from Sequence--->Relations--->ExtendedPhenotype to 
Sequence--->ExtendedPhenotype--->Relations, so the jpa need to know there is relation from ExtendedPhenotype to Relations as well, so it is fully bidirectional relationship now